index.html

```diff
@@ -1,19 +1,63 @@
-<!doctype html>
-
+<!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="utf-8">
+  <head>
+    <link rel="stylesheet" type="text/css" href="css/index.css">
+    <link href="https://fonts.googleapis.com/css?family=Archivo+Narrow|Asap|Cairo|Comfortaa|Hind+Siliguri|Maven+Pro|Signika" rel="stylesheet">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Peter Stone - Resume</title>
+  </head>
+  <body>
+
+    <header>
+      <h1 id="heading">Peter Stone <br> Web Developer</h1>
+      <nav>
+        <a href="home.html">Home</a>
+        <a href="about.html">About</a>
+        <a href="http://www.stonesquareddevelopment.com" target="_blank">Portfolio</a>
+        <a href="contact.html">Contact</a>
```

Comment:
Heads-up on your Home link - it's linking to a non-existent "home.html" file. Perhaps use `href="#"` instead?